crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/ClipSurfaceRemoveArea.shader

```diff
@@ -8,11 +8,42 @@ Shader "Crest/Inputs/Clip Surface/Remove Area"
 {
 	SubShader
 	{
+		Tags { "Queue" = "Geometry" }
+
 		Pass
 		{
 			Blend Off
 			ZWrite Off
 			ColorMask R
+
+			CGPROGRAM
+			#pragma vertex Vert
+			#pragma fragment Frag
+
+			#include "UnityCG.cginc"
+
+			struct Attributes
+			{
+				float3 positionOS : POSITION;
+			};
+
+			struct Varyings
+			{
+				float4 positionCS : SV_POSITION;
+			};
+
+			Varyings Vert(Attributes input)
+			{
+				Varyings o;
+				o.positionCS = UnityObjectToClipPos(input.positionOS);
+				return o;
+			}
+
+			half Frag(Varyings input) : SV_Target
```

Comment:
```
Metal: Error creating pipeline state (Crest/Inputs/Clip Surface/Remove Area): fragment shader color output does not have enough components for the pixel format (MTLPixelFormatBGRA8Unorm_sRGB)
(null)
```
Changing it to `half4` silences this error. It still works with the error though. You can still use `1.0` as return, it's only the return type it cares about.

Reply:
Thanks will change it